doc/maintenance.md

```diff
@@ -137,8 +137,27 @@ $ openssl req -text -noout -in scala-ci.csr
 
 ## Worker offline?
 
-If you see "pending -- (worker) is offline", try waiting ~5 minutes;
-it takes time for ec2-start-stop to spin up workers.
+From the [list of nodes](https://scala-ci.typesafe.com/computer/),
+you can click the entry for the node you're interested in and
+from there, use the "Log" action on the left to see if the node
+is in the process of coming online, and if so, watch it happen.
+
+It takes a few minutes for ec2-start-stop to spin up a worker.
+
+You can manually bring a node online by pressing the "Launch slave
+agent" button.
+
+## Worker goes offline during manual testing?
+
+If you ssh in to a worker to poke around, that won't prevent Jenkins
+from taking the worker offline if it is otherwise idle.  This can get
+annoying.
+
+The only we way know of to force the node to stay up is, ironically,
```

Comment:
There's a secret backdoor to disconnect the worker from Jenkins, while keeping the EC2 node up:  use "relaunch" ad the message when disconnecting. https://github.com/typesafehub/ec2-start-stop/blob/master/src/main/scala/EC2StartStop.scala#L41